### DISCUSSION TITLE
How do I use `log()` with `log_time` without passing in `source_loc`?

### DISCUSSION THREAD

**Discussion by mscofield0** [gabime/spdlog#2136] [Q&A] [answered]
Title says it all.

**Comment by gabime** [comment, marked as answer]
you can't. just pass an empty ```source_loc {}```

**Reply by mscofield0** [reply]
Does an empty `source_loc` get printed still somehow?

**Reply by mscofield0** [reply]
Just tested it out, it doesn't. Thank you gabi
